src/interactive/InfoToggle.jsx

```diff
@@ -20,22 +24,25 @@ export const InfoTooltipTrigger = props => (
 
 const InfoToggle = ({
 	className,
+	children,
+	tooltipContent,
 	label,
-	tooltipId,
+	tooltip,
 	tooltipProps,
-	tooltipContent,
-	onClick,
+	tooltipId,
 	...other
-}) => {
+}: Props) => {
+	const tooltipOptions = {
+		id: tooltipId,
+		...tooltipProps,
+	};
+
 	return (
 		<div className={className} {...other}>
 			<span className="infoToggle-label">{label}</span>
-			<Tooltip
-				id={tooltipId}
-				trigger={<InfoTooltipTrigger onClick={onClick} />}
-				content={tooltipContent}
-				{...tooltipProps}
-			/>
+			<Tooltip trigger={<InfoTooltipTrigger />} {...tooltipOptions}>
+				{tooltipContent || children}
```

Comment:
looks like the interface is ambiguous here - do we prefer `children` over `tooltipContent`? Are we deprecating `tooltipContent` as well?

Reply:
We prefer children here as well